sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -378,6 +379,23 @@ def run_Flatten(self, transform_node):
           PropertyNames.ENCODING: step.encoding,
           PropertyNames.OUTPUT_NAME: PropertyNames.OUT}])
 
+  def apply_WriteToBigQuery(self, transform, pcoll):
+    standard_options = pcoll.pipeline._options.view_as(StandardOptions)
+    if standard_options.streaming:
+      write_truncate = beam.io.BigQueryDisposition.WRITE_TRUNCATE
```

Comment:
Nit: Is this temporary just for line wrapping? If so, I'd just use ()'s in the if.